packages/saved-views-client/src/models/savedViews/View.ts

```diff
@@ -85,7 +85,7 @@ export interface ViewITwinDrawing extends ViewITwin2d {
     [qx: number, qy: number, qz: number, ax: number],
     [qx: number, qy: number, qz: number, ax: number],
   ];
-  modelExtents: [
+  modelExtents?: [
```

Comment:
Don't forget to add a changelog entry to CHANGELOG.md

Reply:
@roluk , added a changelog entry.